src/Compilers/VisualBasic/Portable/BoundTree/BoundInterpolatedStringExpression.vb

```diff
@@ -36,6 +36,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 #If DEBUG Then
         Private Sub Validate()
             Debug.Assert(Type.SpecialType = SpecialType.System_String)
+            Debug.Assert(Not Contents.Where(Function(content) content.Kind <> BoundKind.Interpolation AndAlso content.Kind <> BoundKind.Literal).Any())
```

Comment:
Consider simplifying: `Contents.All(Function(content) content.Kind = Interpolation OrElse content.Kind = Literal)`

Reply:
Ping just to make sure you didn't miss this comment. Thanks

Reply:
> Ping just to make sure you didn't miss this comment.

Opened https://github.com/dotnet/roslyn/issues/22869